README.md

```diff
@@ -192,33 +198,59 @@ The following instructions show you how to connect the API to an Oracle database
         $ git submodule update --init
         ```
 
-5. Copy [db/oracledb-example.js](db/oracledb-example.js) to `db/db.js` and modify as necessary:
+5. Copy [db/oracledb-example.js](db/oracledb-example.js) to `db/oracledb.js` and modify as necessary:
 
     ```shell
-    $ git mv --force db/oracledb-example.js db/db.js
+    $ git mv db/oracledb-example.js db/oracledb.js
+    ```
+
+6. Make sure to require the correct path for the new db file at [resources files](resources/api-resources.js#L3):
+
+    ```js
+    const db = appRoot.require('/db/oracledb');
     ```
 
 ## Docker
 
 [Dockerfile](Dockerfile) is also provided. To run the app in a container, install [Docker](https://www.docker.com/) first, then:
 
-1. If the API is required [node-oracledb](https://oracle.github.io/node-oracledb/) to connect to an Oracle database, download an [Oracle Instant Client 12.2 Basic Light zip (64 bits)](http://www.oracle.com/technetwork/topics/linuxx86-64soft-092277.html) and place into `./bin` folder.
+1. Modify `WORKDIR` from the [Dockerfile](Dockerfile#L4-L5):
+
+    ```Dockerfile
+    # Copy folder to workspace
+    WORKDIR /usr/src/<my-api>
+    COPY . /usr/src/<my-api>
+    ```
+
+2. If the API is required [node-oracledb](https://oracle.github.io/node-oracledb/) to connect to an Oracle database, download an [Oracle Instant Client 12.2 Basic Light zip (64 bits)](http://www.oracle.com/technetwork/topics/linuxx86-64soft-092277.html) and place into `./bin` folder. In addition, uncomment [the following codes](Dockerfile#L8-L15) from the Dockerfile:
```

Comment:
nitpick: this should be "the following code" or "the following lines". "code" usually isn't plural in the context of computer programming code

Reply:
Good to know, thanks.